underfs/s3/src/main/java/alluxio/underfs/s3/S3Utils.java

```diff
@@ -22,34 +22,42 @@
  */
 public final class S3Utils {
```

Comment:
Hmm, those constants and interface between AWS SDK and jets3t are different, which makes it a bit hard to consolidate. Yeah, we can refactor this later.